### DISCUSSION TITLE
Extra costs when having multiple websites with different domain?

### DISCUSSION THREAD

**Discussion by SquidXTV** [community/community#75614] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi, quick question:

I know that I can create a website using GitHub pages and if I want multiple I just create multiple repositories and can set up them differently.

My question now, do I need to pay extra or something if having a second GitHub pages repository?
I somehow remember that GitHub Pro gives access to only one GitHub pages website, is this correct?

**Comment by SquidXTV** [comment]
Also another question:
I am just seeing this [prohibited uses](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#prohibited-uses), and it is talking about selling stuff on a GitHub hosted website.

Would a DJ - Portfolio with contact information and maybe a list of services add to that?
The transactions wouldn't be done through the page but differently.

**Reply by elderguardian** [reply]
> shouldn't be used for sensitive transactions like sending passwords or credit card numbers.

Just follow that, it will be okay

**Comment by elderguardian** [comment, marked as answer]
I also have multiple gh pages and I never paid anything

**Reply by SquidXTV** [reply]
ok thanks for your help
